templates/common/on-prem/files/coredns.yaml

```diff
@@ -19,7 +19,7 @@ contents:
           path: "/etc/kubernetes/static-pod-resources/coredns"
       - name: kubeconfig
         hostPath:
-          path: "/etc/kubernetes/kubeconfig"
```

Comment:
Are the two changes of updating the path of the kubeconfig file and removing the mdns plugin related? If not they should probably go in separate PRs or at least separate commits.

Reply:
Yes, in order for the coredns-monitor to read the node list and populate this new template we need the kubelet kubeconfig. The old one doesn't have permission to read nodes.